views/user/story-create.hbs

```diff
@@ -1,15 +1,21 @@
 <h2>Create a story</h2>
- 
+
 <form action="/create" method="post">
-<div class="textarea">
- 
-  <label for="title-input">Title:</label>
-  <input type="text" name="title" id="title-input" />
- 
-  <label for="description-input">My story:</label>
-  <textarea name="description" id="description-input" cols="150" rows="25"> </textarea>
- 
-  <button class="btn btn-secondary" type="submit">Create</button>
- </div>
-</form>
+  <div class="textarea">
+
+    <label for="title-input">Title:</label>
+    <input type="text" name="title" id="title-input" />
```

Comment:
I see this field is required in the Model. So you can also add some simple form validation here by adding the `required` property on the input